### ISSUE TITLE
Error installing Tensorflow in R Studio

### ISSUE THREAD

**Issue by marireeves** [rstudio/keras#916]
Hi There I'm getting the following error when I use the install_tensorflow() function. 

"Error: Error 1 occurred installing packages into conda environment r-reticulate
In addition: Warning message:
In normalizePath(path.expand(path), winslash, mustWork) :
  path[1]="C:\Users\marireeves\AppData\Local\Continuum\anaconda3\envs\rstudio/python.exe": The system cannot find the file specified"

I can see the typo in the final slash in that path, but don't know to fix it, so I'm reporting it.
I'm running:

Win 10
R 3.6.1
R Studio (Version 1.2.1335) R

**Comment by dfalbel** [comment]
What version of  `reticulate` are you using? I remember fixing something like this recently.

**Comment by marireeves** [comment]
version 1.13

**Comment by sunwanwan321** [comment]
Hi, I have met the same Error when I install "tensorflow",but Ihave slove the Error through:
1. install "Anaconda" with the newest version in my computer,and then set it into "r-reticulate";
2.restart my computer
3.install.packages("tensorflow")
devtools::install_github("rstudio/tensorflow")
install_tensorflow()
library(tensorflow)
library(keras)
finally, it works, you can try the procedures, good luck

**Comment by t-kalinowski** [comment]
Hello,

The code that installs tensorflow and keras has been [refactored](https://github.com/rstudio/keras/pull/1251).
Please update to the latest versions of reticulate, tensorflow, and keras:
```R
install.packages("remotes")
remotes::install_github(paste0("rstudio/", c("reticulate", "tensorflow", "keras")))
reticulate::install_miniconda() # skip this if you want to self-install conda or use venv
keras::install_keras()
```

Please let us know if after updating you are still encountering installation issues:
https://github.com/rstudio/tensorflow/issues/new
